h2/src/main/org/h2/command/dml/Select.java

```diff
@@ -569,7 +569,7 @@ protected ResultInterface queryWithoutCache(int maxRows, ResultTarget target) {
                 limitRows = Math.min(l, limitRows);
             }
         }
-        boolean lazy = session.isLazyQueryExecution() &&
+        lazy = session.isLazyQueryExecution() &&
```

Comment:
Since methods `query` and `queryWithoutCache` have parameters that affect the value of `lazy`, it does not look like correct fix.

Reply:
If it was a wrong fix then there are not enough tests in testing framework, because all tests passed with this incorrect change.

Reply:
There are never enough tests in any complex projects, feel free to add more :)